tests/chart/values.yaml

```diff
@@ -139,9 +142,9 @@ nodeAgent:
 clamav:
   name: clamav
   image:
-    repository: quay.io/armosec/klamav
-    tag: beta5
-    pullPolicy: Always
+    repository: clamav/clamav-debian
```

Comment:
Please wrap this image using the [script](https://github.com/armosec/kubecop/tree/main/resources/clamav) because we filter out signatures that isn't for Linux